bpf-gpl/tc.c

```diff
@@ -106,7 +106,7 @@ int calico_tc_norm_pol_tail(struct __sk_buff *skb)
 	}
 
 	state->pol_rc = execute_policy_norm(skb, state->ip_proto, state->ip_src,
-					    state->ip_dst, state->sport, state->dport);
+					    state->ip_dst, state->sport, state->dst.u.port);
```

Comment:
this is probably gonna get revisited and we will just pass the state, otherwise would perhaps make some sense to reuse both ports rather than a single 16-bit value for 2 8-bit ones. But I had the impression that you want to use the other for some additional id, right?

Reply:
Using ID in place of src port was for conntrack, if we are to do it like how linux does